quipucords/scanner/network/runner/roles/redhat_release/tasks/main.yml

```diff
@@ -4,25 +4,17 @@
   set_fact:
     internal_host_started_processing_role: "redhat_release"
 
-- name: initialize redhat_release facts
-  set_fact:
-    redhat_release_name: ''
-    redhat_release_version: ''
-    redhat_release_release: ''
-  ignore_errors: yes
-
 - name: gather internal_redhat_release fact
   raw: rpm -q --queryformat "%{NAME}\n%{VERSION}\n%{RELEASE}\n" --whatprovides redhat-release
   register: internal_redhat_release
```

Comment:
why did you change it to true instead of yes ?
It's a bit confussing but I guess Ansible standard is yes instead of true.
https://docs.ansible.com/ansible/latest/user_guide/playbooks_error_handling.html#

Reply:
awesome!!